packages/solana/package.json

```diff
@@ -14,7 +14,8 @@
     "test": "vitest"
   },
   "dependencies": {
-    "@chainflip/utils": "0.4.0"
+    "@chainflip/utils": "0.4.0",
+    "@solana/web3.js": "^1.95.8"
```

Comment:
this is the not affected version right. just checking :D

Reply:
no, but i actually managed to install the affected version when i started with this pr and was wondering why the CI does not work 😅